frontend/ui/widget/screenshoter.lua

```diff
@@ -47,11 +48,20 @@ function Screenshoter:getScreenshotDir()
 end
 
 function Screenshoter:onScreenshot(screenshot_name, caller_callback)
+    local prefix = self.prefix
+    local file = self.ui.document and self.ui.document.file -- currently opened book
+    if file then
+        local curr_page = "p" .. self.ui:getCurrentPage()
+        if self.ui.pagemap and self.ui.pagemap:wantsPageLabels() then
+            curr_page = self.ui.pagemap:getCurrentPageLabel(true)
```

Comment:
Shouldn't it also get the "p" prefix with page labels ?
(Just mentionning it if this was an oversight and consistency is welcome, no strong opinion.)

Reply:
As `p_label` that's fine but as `plabel` that doesn't work.

Reply:
Most often, the label is a number - in the worst case, it's a roman number, so `piii` or _pXI_ (which may indeed look odd). Anyway, not minding either choice.

Reply:
I want it either this way or the way it was before, but between the two I don't really mind.

Reply:
```suggestion
        local curr_page = "p" .. self.ui:getCurrentPage()
        if self.ui.pagemap and self.ui.pagemap:wantsPageLabels() then
            curr_page = "p_" .. self.ui.pagemap:getCurrentPageLabel(true)
```